core/new-gui/src/app/workspace/service/undo-redo/undo-redo.service.ts

```diff
@@ -53,6 +56,8 @@ export class UndoRedoService {
       this.canUndoStream.next(this.canUndo());
 
       console.log("service can undo", this.canUndo());
+      const commandMessage: CommandMessage = { action: "undoredo", parameters: [], type: "undo" };
+      this.sendCommand(JSON.stringify(commandMessage));
```

Comment:
can you change the line 58 log to be under those two commends? we print such "success" log only when it actually "succeeded".

Reply:
Done.